packages/api/models/schemas.js

```diff
@@ -142,6 +142,11 @@ module.exports.collection = {
             description: 'Folder used to save the granule in the bucket. '
                          + 'Defaults to the collection url path',
             type: 'string'
+          },
```

Comment:
We're adding "fileType" to collection here to allow collection configuration combined with DiscoverGranules to add a fileType to the granules->files->fileType on ingest, in similar fashion to what's going on in kinesis/sns ingest and with ParsePDR injecting those values. 

This value should only be use by core to populate the value on discovery, however the value is not constrained to CNM types as core behavior for unrecognized/missing types is by design currently to convert to 'data' for CMR publication and users may want to specify other fileTypes now/in the future.

Reply:
The way I read this, we're not adding any kind of restraint on the `fileType` that a user can configure. What we _are_ doing is specifying a subset of `fileType` values (that correspond to CNM `fileType`s) that we will support with our provided tasks - `discoverGranules`, `postToCMR`, etc. Is this correct?

Reply:
@ifestus  exactly, yes.

Reply:
It's tricky.  The granule->file schema is independent from the behavior of the post to CMR *task*.   In fact, people may not want to post to CMR.    This is part of that documentation of components thing I was bringing up during the Wednesday meeting.